### DISCUSSION TITLE
Add featured game to your miniprofile

### DISCUSSION THREAD

**Discussion by gamer2810** [gamer2810/steam-miniprofile#21] [Announcements]
By adding `appId` query parameter, you can now feature your favorite game with its playtime on your miniprofile!
![image](https://github.com/user-attachments/assets/ce73179a-0be0-40a1-9500-43b03514357f)

<hr /><em>This discussion was created from the release <a href='https://github.com/gamer2810/steam-miniprofile/releases/tag/featured-game'>Add featured game to your miniprofile</a>.</em>

**Comment by huytc** [comment]
troi` di lam di 3
